app/models/preference.rb

```diff
@@ -1,2 +1,21 @@
+# == Schema Information
+#
+# Table name: preferences
+#
+#  id          :bigint           not null, primary key
+#  user_id     :bigint           not null
+#  name        :string
+#  description :text
+#  restriction :boolean          default(TRUE), not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_preferences_on_user_id  (user_id)
+#
 class Preference < ApplicationRecord
+  MAX_PREFERENCES = 10
```

Comment:
This is not in the scope of this ticket so you can leave it like this and tackle later, but I believe the limit is 5 😅